UnityProject/Assets/Strings/tips/GeneralTips.asset

```diff
@@ -0,0 +1,65 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: 12b945c0690aa6045ae7abb331ebfe26, type: 3}
+  m_Name: GeneralTips
+  m_EditorClassIdentifier: 
+  strings:
+  - Alt-click while holding a health analyzer to get a full report on the body part
+    you're targeting plus the organs inside of it.
+  - All command roles and the clown have their own special rubber stamps
```

Comment:
```suggestion
  - All command roles and the clown have their own special rubber stamps.
```

Reply:
this makes me wonder, do we have a lot of tips in here that are not strictly speaking true, yet? We don't actually have stamps in game, and I'd like to not put in tips that do not have the intended gameplay backing them at the moment.

Reply:
I agree. Do we have stamp objects, at least?

Reply:
idk if we have any existing prefabs in the project, but we definitely don't have any mapped into any mainstations.